doc/source/whatsnew/v1.0.0.rst

```diff
@@ -253,6 +253,9 @@ Other API changes
   See :ref:`units registration <whatsnew_1000.matplotlib_units>` for more.
 - :meth:`Series.dropna` has dropped its ``**kwargs`` argument in favor of a single ``how`` parameter.
   Supplying anything else than ``how`` to ``**kwargs`` raised a ``TypeError`` previously (:issue:`29388`)
+- Initialising an empty :class:`pandas.core.series.Series` without specifying a dtype will raise a FutureWarning now.
```

Comment:
can you make this a sub-section in api breaking changes section. this is a major change and needs to be shown as such.